src/controllers/formExampleController.ts

```diff
@@ -22,14 +23,15 @@ export class FormExampleController {
 
   public async post(req, res, next) {
     let formExampleModel = new FormExampleModel(
-      req.body.fullName,
+      lettersAndSpaceInputString(req.body.fullName),
```

Comment:
Just a thought, should we move these sanitisation methods into the FormExampleModel constructor? Means that every usage of this class will be sanitised by default.